.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "contracts/organization"]
+	path = contracts/organization
+	url = https://github.com/OpenST/organization-contracts.git
```

Comment:
Do you think similar below script is also needed here for updating the submodule? I am not 100% sure 😕 
https://github.com/OpenST/openst-contracts/blob/develop/package.json#L28
```
"update": "git submodule update --init --recursive
```

Reply:
There is not package.json file in this PR, that will be added with another ticket.